addons/mail/views/mail_canned_response_views.xml

```diff
@@ -9,8 +9,11 @@
                 <search string="Canned Responses Search">
                     <field name="source"/>
                     <field name="substitution"/>
-                    <filter string="My canned responses" name="filter_create_uid" domain="[('create_uid', '=', uid)]"/>
-                    <filter string="Shared canned responses" name="filter_is_shared" domain="[('is_shared', '=', True)]"/>
+                    <filter string="Private" name="filter_create_uid" domain="[('create_uid', '=', uid), ('is_shared', '=', False)]"/>
```

Comment:
I think we can remove the create_uid part, the filter stays "private" so it should be all is_shared = False.

"create_uid" would only make sense if the filter was "my canned response".

If a db admin/system can see all of them, I'd expect Private to list all of the private one regardless of user.